### DISCUSSION TITLE
Welcome to destreamer Discussions!

### DISCUSSION THREAD

**Discussion by snobu** [snobu/destreamer#321] [General]
Not sure what we do with "Discussions", but if GitHub has it, let's have it.

**Comment by coccodavid** [comment]
Hi!
My professor started to use "Panopto" streaming. Is Destreamer available also for this streaming service?
If not, can you make it please? :D thanks for your work guys

**Comment by aazish** [comment]
> Not sure what we do with "Discussions", but if GitHub has it, let's have it.

Sir, can you please provide me an exe file with MS Edge path instead of Chromium as we have organization restrictions to use only MS Edge. I'm not developer and I tried my best to make exe but couldn't succeed? Please
